docs/reference/commandline/login.md

```diff
@@ -43,7 +43,7 @@ You can log into any public or private repository for which you have
 credentials.  When you log in, the command stores encoded credentials in
 `$HOME/.docker/config.json` on Linux or `%USERPROFILE%/.docker/config.json` on Windows.
 
-## Credentials store
+## <a name="credentials-store">Credentials store</a>
```

Comment:
No, it's not needed, it's automatically generated when converting the Markdown to HTML (including when reading on GitHub)